src/pages/app-about/app-about.scss

```diff
@@ -124,6 +124,18 @@
       max-width: 420px;
       text-align: left;
     }
+
+    button {
+      &:hover {
+        background: #ff5d00;
+        border-color: white;
+      }
+
+      &:not(:disabled):active {
+        background-color: #ff5d00;
+        border-color: white;
```

Comment:
There is a global variable for white, we should use that instead incase we ever need to adjust the shade of it at any point.

Reply:
you are right, fixed